app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.bottlerocketstudios.android_maps_demo">
```

Comment:
@Josh-Gebbeken Suggest using `com.bottlerocketstudios.mapsdemo` to match applicationId